pyface/action/api.py

```diff
@@ -9,6 +9,68 @@
 # Thanks for using Enthought open source!
 
 
+"""
+
+API for the ``pyface.action`` submodule.
+
+Action and subclasses
+---------------------
+
+- :class:`~.Action`
+- :class:`~.FieldAction`
+- :class:`~.GUIApplicationAction`
+- :class:`~.ListeningAction`
+- :class:`~.TraitsUIWidgetAction`
+- :class:`~.WindowAction`
+
+Action Controller
+-----------------
+
+- :class:`~.ActionController`
+
+Action Event
+------------
+
+- :class:`~.ActionEvent`
+
+Action Manager and subclasses
+-----------------------------
+
+- :class:`~.ActionManager`
+
+Toolkit-specific subclasses:
```

Comment:
Not sure my understanding of "toolkit-specific" is the same as what you are intending.
If a class is toolkit-specific, then I need to know which toolkit it is designed for, e.g. MenuManager may only support Qt and I need to know that. But I think these classes are aiming at a toolkit independent API. And ToolPaletteManager only has a Wx implementation.

I am guessing you mean these classes depend on one GUI toolkit, but not down to the specific of which one.

Reply:
> I am guessing you mean these classes depend on one GUI toolkit, but not down to the specific of which one.

Yes. That's what I meant. But you're right - that's a implementation detail and clearly, it shouldn't be mentioned in the api! Removed this implementation detail.